dynawo/sources/Modeler/Common/DYNSubModel.cpp

```diff
@@ -1310,6 +1316,24 @@ SubModel::getSubModelParameterValue(const string& nameParameter, double& value,
   }
 }
 
+void
+SubModel::initializeLocalInitParameters(boost::shared_ptr<parameters::ParametersSet> intputlocalInitParams) {
+  if (intputlocalInitParams->hasParameter("mxiter"))
```

Comment:
Should be in the solver part

Reply:
You mean this function should be in SolverKINSubModel ?